### PR TITLE
Add Wolvesville

### PR DESCRIPTION
**Pull request by down-bad**
Wolvesville is an online multiplayer game. This presence should support all Wolvesville sites, including the game.

Screenshot 2021-09-25 192429
Screenshot 2021-09-25 192518
Screenshot 2021-09-25 192606
Screenshot 2021-09-25 192702

Game:
Screenshot 2021-09-25 191910
Screenshot 2021-09-25 192050
Screenshot 2021-09-25 192120
Screenshot 2021-09-25 192211
Screenshot 2021-09-25 192346

### CODE REVIEW

File: websites/W/Wolvesville/presence.ts [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+    data: PresenceData = {
+      largeImageKey: logoArr[logo] || "wov",
+      startTimestamp: startedTime
+    };
+
+  if (document.location.href.includes("blog.wolvesville.com")) { //Wolvesville Blog
+    data.smallImageKey = "wov_blog";
+    data.smallImageText = "Development Blog";
+    if (document.location.pathname === '/' || document.location.pathname.startsWith('/page')) {
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:21]:
```suggestion
    if (document.location.pathname === "/" || document.location.pathname.startsWith("/page")) {
```

File: websites/W/Wolvesville/presence.ts [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+            "div.css-1dbjc4n.r-1j16mh1.r-1d6rzhh.r-sga3zk.r-12c3ph5.r-1sbahrg.r-lrvibr.r-7a29px > div.css-1dbjc4n.r-1awozwy.r-1pi2tsx.r-1777fci.r-13qz1uu > div.css-901oao.r-1281ybr"
+          ).textContent;
+        }
+      } else {
+        lobbyChar = document.querySelector(
+          "div.css-1dbjc4n.r-1j16mh1.r-1d6rzhh.r-1loqt21.r-sga3zk.r-1sbahrg.r-1otgn73.r-lrvibr.r-7a29px > div.css-1dbjc4n.r-1awozwy.r-1pi2tsx.r-1777fci.r-13qz1uu > div.css-901oao"
+        ).textContent;
+      }
+
+      if (preGameLobby && lobbyChar === '') {
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:217]:
```suggestion
      if (preGameLobby && lobbyChar === "") {
```

File: websites/W/Wolvesville/presence.ts [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+      )
+        ? true : false;
+
+      if (!preGameLobby) {
+        preGameLobby = document.querySelector(
+          "div.css-1dbjc4n.r-1j16mh1.r-1d6rzhh.r-sga3zk.r-12c3ph5.r-1sbahrg.r-lrvibr.r-7a29px > div.css-1dbjc4n.r-1awozwy.r-1pi2tsx.r-1777fci.r-13qz1uu > div.css-901oao.r-1281ybr"
+        )
+          ? true : false;
+
+        var lobbyChar;
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:205]:
```suggestion
```

File: websites/W/Wolvesville/presence.ts [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+        if (preGameLobby) {
+          lobbyChar = document.querySelector(
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:207]:
```suggestion
          const lobbyChar = document.querySelector(
```

**Review by EncryptedDev — DISMISSED** [review]

File: websites/W/Wolvesville/presence.ts (hunk 4 above)
Comment (down-bad) [reply websites/W/Wolvesville/presence.ts:207]: i cant do this because then it cant read the const in line 216

Comment (EncryptedDev) [reply websites/W/Wolvesville/presence.ts:207]: Sorry, didn't see that - use `let` instead of `var`

**Review by Slowlife01 — DISMISSED** [review]

File: websites/W/Wolvesville/presence.ts [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+      const member = document.querySelector("#staff_member_name").textContent;
+      data.state = `Reading about ${member}`;
+    }
+  } else if (document.location.href.includes("voting.wolvesville.com")) { //Voting Gallery
+    data.details = "Voting Gallery";
+
+    const submissionView = document.querySelector(
+      ".css-757v71"
+    )
+      ? true : false;
```
Comment (Slowlife01) [inline websites/W/Wolvesville/presence.ts:76]: You could do something like this to make it more clean
```js
    const submissionView = !!document.querySelector(
      ".css-757v71"
    );
```

Comment (down-bad) [reply websites/W/Wolvesville/presence.ts:76]: oh great, thanks

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**